include/records.hrl

```diff
@@ -32,6 +32,10 @@
           auth_plugin_data :: binary()
          }).
 
+-record(auth_more_data, {
+          message :: term()
+         }).
```

Comment:
This doesn't seem useful. Can it be a type instead: `-type auth_more_data :: fast_auth_completed | full_auth_requested | {public_key, _}.`?

Btw, the record definitions which are only used in one module are probably better to define there than in this include file, but that's another PR perhaps...

Reply:
You're right, this is way better :) Changed as suggested in last commit.